src/aks-preview/azext_aks_preview/_params.py

```diff
@@ -2135,6 +2135,26 @@ def load_arguments(self, _):
             "revision", validator=validate_azure_service_mesh_revision, required=True
         )
 
+    with self.argument_context("aks mesh upgrade rollback") as c:
+        c.argument(
+            "yes",
+            options_list=["--yes", "-y"],
+            help="Do not prompt for confirmation.",
+            action="store_true",
+            default=False,
+            required=False
```

Comment:
I think you may omit `required=False` as this is the default behavior

Reply:
I am running into this error on removing `required=False`
`the following arguments are required: --yes/-y`

Reply:
OK, if you set the default value to `False` in custom.py then you could probably remove the `default=False,
required=False` here.